<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>447</width>
    <height>245</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SHUTDOWN TIMER</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>461</width>
      <height>361</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #fff</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="shutdown">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>180</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
     background-color: #233738;
	border: none;
color: #EEEEEE;
     border-radius: 3px;



}



QPushButton:hover{
   background-color:#EEEEEE;
	color: #233738;
     border-radius: 3px;
	border: 2px solid #233738;

}



QPushButton:pressed {
       
 border: 3px solid #233738;
     border-radius:3px;
}</string>
     </property>
     <property name="text">
      <string>START TIMER</string>
     </property>
    </widget>
    <widget class="QFrame" name="frame">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>451</width>
       <height>261</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
   background-color:#EEEEEE;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>140</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #233738;
    background-color:  none;</string>
      </property>
      <property name="text">
       <string>  MINUTES:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #233738;
    background-color:  none;
</string>
      </property>
      <property name="text">
       <string>HOURS:</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="hBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QSpinBox{
	 background-color: #233738;
		border: none;
		color: #FBF0F0;
	border-radius: 5px;
border: 2px solid #323232
	
}</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="maximum">
       <number>23</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="mBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>180</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QSpinBox{
	 background-color: #233738;
		color: #FBF0F0;
	border-radius: 5px;
border: 2px solid #323232
	
}</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="maximum">
       <number>59</number>
      </property>
      <property name="value">
       <number>1</number>
      </property>
     </widget>
     <widget class="QLabel" name="last_time">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>150</y>
        <width>271</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #233738;
    background-color:  none;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="stop">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>180</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
     background-color: #233738;
	border: none;
color: #EEEEEE;
     border-radius: 3px;



}



QPushButton:hover{
   background-color:#EEEEEE;
	color: #233738;
     border-radius: 3px;
	border: 2px solid #233738;

}



QPushButton:pressed {
       
 border: 3px solid #233738;
     border-radius:3px;
}</string>
     </property>
     <property name="text">
      <string>STOP</string>
     </property>
    </widget>
    <widget class="QProgressBar" name="progressBar">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>421</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QProgressBar::chunk {
   background-color: qlineargradient(spread:pad, x1:0 y1:0, x2:1 y2:0, stop:0 #000000, stop:1 #233738);
    border-radius: 2px;
   
 }

 QProgressBar {
   background: #EEEEEE;

	border: 1px solid rgb(33,33,33);
     border-radius: 4px;
     padding: 3px;
border-width: 2px;



 }</string>
     </property>
     <property name="maximum">
      <number>999999999</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="textVisible">
      <bool>false</bool>
     </property>
    </widget>
    <zorder>frame</zorder>
    <zorder>shutdown</zorder>
    <zorder>stop</zorder>
    <zorder>progressBar</zorder>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
